.github/workflows/build_publish.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and publish the release
+
+on:
+  release:
+    types: [ released ]
+
+jobs:
+  test:
+    name: Run checks
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - uses: Gr1N/setup-poetry@v2
+        with:
+          poetry-version: 1.0.10
+      - run: poetry install
+      - run: poetry run poe test
+      - run: poetry run poe typecheck
+      - run: poetry run poe codestyle
+      - run: poetry run poe liccheck
+
+  build:
```

Comment:
Can't we somehow avoid having duplicate definitions of `test` and `build` jobs in both workflow files?
Maybe `publish` can be enabled conditionally using `if:`?

Reply:
ah, maybe, yeah (!)

Reply:
done